readthedocs/projects/views/public.py

```diff
@@ -367,7 +369,7 @@ def project_versions(request, project_slug):
     version_filter = request.GET.get('version_filter', '')
     if version_filter:
         inactive_versions = inactive_versions.filter(verbose_name__icontains=version_filter)
-    inactive_versions = inactive_versions[:100]
+    inactive_versions = inactive_versions[:max_inactive_versions]
```

Comment:
Feels like the length check should be done here, before we truncate it. Otherwise won't the template never show additional versions because it's already cut off at `max_inactive_versions` total?

Reply:
The message is shown because there will be 100 inactive versions and the check checks `>=`. It is an extra query to do a count but I don't think it'll be too costly.